advocacy_docs/supported-open-source/pgbackrest/04-recommended_settings.mdx

```diff
@@ -78,14 +78,32 @@ WARN: unable to check pg1: [VersionNotSupportedError] unexpected control version
 
 To solve this problem, add `pg-version-force=15` into your configuration, in the stanza section (where `15` is the major release version used).
 
+### EDB Transparent Data Encryption support
+
+Transparent data encryption (TDE) is an optional feature supported by EDB Postgres Advanced Server and EDB Postgres Extended Server.
+
+To let pgBackRest work when this feature in enabled, add the following settings to the `[global]` section:
```

Comment:
```suggestion
To enable this feature in pgBackRest , add the following settings to the `[global]` section:
```

Reply:
Thank you very much for the review and the rewording suggestion @drothery-edb!